s3_sequins_test.go

```diff
@@ -73,7 +70,46 @@ func TestS3Backend(t *testing.T) {
 	assert.Equal(t, 20, len(files), "the list of files should be correct")
 }
 
+func putS3(svc *s3.S3, bucket, dst, src string) error {
+	data, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer data.Close()
+
+	params := &s3.PutObjectInput{
+		Bucket:		aws.String(bucket),
+		Key:		aws.String(dst),
+		Body:		data,
+	}
+
+	_, err = svc.PutObject(params)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func putS3Blob(svc *s3.S3, bucket, dst string, src []byte) error {
```

Comment:
If you make this take an `io.ReadSeeker` or whatever `PutObjectInput` wants, then you can reuse it in the above method.